akita/__tests__/filters-plugin.spec.ts

```diff
@@ -0,0 +1,291 @@
+import { skip, take } from 'rxjs/operators';
+import { createWidget, createWidgetCompleted, WidgetsQuery, WidgetsStore } from './setup';
```

Comment:
Please apply prettier to all files.

Reply:
I thought it was applied after each commit. But I applied Prettier.